script/testing/oltpbench/test_oltpbench.py

```diff
@@ -74,10 +85,12 @@ def run_pre_test(self):
         self.install_oltp()
         self.config_xml_file()
         self.create_result_dir()
+        self.create_summary_dir()
```

Comment:
Just to clarify the results are the raw stdout and the summary is the nice json?

Reply:
You are correct.